src/test/java/org/apache/drillui/test/framework/testng/secure/login/LoginTest.java

```diff
@@ -16,23 +16,70 @@
  */
 package org.apache.drillui.test.framework.testng.secure.login;
 
-import org.apache.drillui.test.framework.initial.TestProperties;
+import org.apache.drillui.test.framework.initial.PropertiesConst;
+import org.apache.drillui.test.framework.steps.webui.AuthSteps;
+import org.apache.drillui.test.framework.steps.webui.ProfilesSteps;
+import org.apache.drillui.test.framework.steps.webui.QueryResultsSteps;
 import org.apache.drillui.test.framework.testng.secure.BaseSecureTest;
 import org.testng.annotations.Test;
-import org.apache.drillui.test.framework.steps.webui.AuthSteps;
-
-import static org.testng.Assert.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertFalse;
 
 public class LoginTest extends BaseSecureTest {
+  private AuthSteps AuthSteps = new AuthSteps();
+  private QueryResultsSteps QueryResultsSteps =  new QueryResultsSteps();
+  private ProfilesSteps ProfilesSteps = new ProfilesSteps();
```

Comment:
A field name should begin with a lowercase letter.

Reply:
Done